core/tests/protractor_utils/SkillEditorPage.js

```diff
@@ -140,18 +140,15 @@ var SkillEditorPage = function() {
     await this.selectDifficultyForRubric(difficulty);
     var addRubricExplanationButton = element(
       by.css('.protractor-test-add-explanation-button-' + difficulty));
-    await waitFor.elementToBeClickable(
-      addRubricExplanationButton,
-      'Add Rubric Explanation button takes too long to be clickable');
-    await addRubricExplanationButton.click();
+    await action.click(
+      'Add Rubric Explanation Button', addRubricExplanationButton);
     await waitFor.visibilityOf(
       rubricExplanationEditorElement,
       'Rubric explanation editor takes too long to appear');
-    await (await browser.switchTo().activeElement()).sendKeys(explanation);
-    await waitFor.elementToBeClickable(
-      saveRubricExplanationButton,
-      'Save Rubric Explanation button takes too long to be clickable');
-    await saveRubricExplanationButton.click();
+    var activeExplanation = await browser.switchTo().activeElement();
+    await action.sendKeys('Explanation Field', activeExplanation, explanation);
```

Comment:
This won't work. As described in the last bullet under ["Anti-Patterns" on our wiki page](https://github.com/oppia/oppia/wiki/End-to-End-Tests#anti-patterns), passing the result of `browser.switchTo().activeElement()` into our `action` functions results in this error:

```
Failed: Cannot read property 'bind' of undefined
```

That error is now showing up in your CI runs: https://github.com/oppia/oppia/pull/13930/checks?check_run_id=3744875614.

Instead, you need to use `element(by.css(...))` to select the element that `browser.switchTo().activeElement()` is finding. The wiki page linked above describes how to do this

Reply:
Sure will update it.